### PR TITLE
Docker Explorer (not view) 

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

**Comment by ghogen** [comment]
@BigMorty @karolz-ms

**Review by BigMorty — APPROVED** [review]
The changes look great, thanks. One thing I noticed was we refer to the panes in the Explorer as "panes" and sometimes "sections". I refer to them as "panels". We should sync on this, "panes" is probably best.

File: docs/containers/overview.md [hunk 1]
```diff
@@ -37,17 +37,17 @@ You can add Docker files to your workspace by opening the Command Palette (`kb(w
 
 The extension recognizes workspaces that use most popular development languages (C#, Node.js, Python, Ruby, Go, and Java) and customizes generated Docker files accordingly.
 
-## Docker view
+## Docker Explorer
 
-The Docker extension contributes a Docker view to VS Code. The Docker view lets you examine and manage Docker assets: containers, images, volumes, networks, and container registries. If the Azure Account extension is installed, you can browse your Azure Container Registries as well.
+The Docker extension contributes a Docker Explorer to VS Code. The Docker Explorer lets you examine and manage Docker assets: containers, images, volumes, networks, and container registries. If the Azure Account extension is installed, you can browse your Azure Container Registries as well.
```
Comment (karolz-ms) [inline docs/containers/overview.md:42]:
```suggestion
The Docker extension contributes a Docker Explorer view to VS Code. The Docker Explorer lets you examine and manage Docker assets: containers, images, volumes, networks, and container registries. If the Azure Account extension is installed, you can browse your Azure Container Registries as well.
```

**Review by karolz-ms — APPROVED** [review]
Looks great, thanks Gordon! Just one suggestion to consider--HTH

**Pull request merged**